reporter-web-app/build.gradle

```diff
@@ -5,7 +5,11 @@ plugins {
 node {
     version = '8.11.4'
     yarnVersion = '1.9.4'
-
+    // Installing NPM is needed for installing Yarn because
+    // the Gradle Node plugin install Yarn via NPM.
+    npmVersion = '6.4.0'
```

Comment:
Shouldn't we use the latest `6.4.1`?

Reply:
@fviernau and I agreed to use 6.4.0 to align with what we currently use in `.appveyor.yml` and `.travis.yml`.